Core/ShopControl.php

```diff
@@ -102,6 +108,10 @@ public function start($sClass = null, $sFunction = null, $aParams = null, $aView
                 $this->blockRequest();
             }
 
+            if(Registry::getConfig()->getConfigParam(self::BLOCK_ONLY_BLOCKLIST) === true) {
+                return;
```

Comment:
Wenn der return hier kommt, würde der nie in die parent::start springen.

Reply:
Gutes Auge, danke. Sowas kommt wenn man nicht getestet hat